addons/base_vat_autocomplete/models/res_partner.py

```diff
@@ -42,19 +42,15 @@ def _check_city(lines, country='BE'):
         for partner in self:
             if not partner.vat:
                 return {}
-            if len(partner.vat) > 5 and partner.vat[:2].lower() in stdnum_vat.country_codes:
-                # Equivalent to stdnum_vat.check_vies(partner.vat).
-                # However, we want to add a custom timeout to the suds.client
-                # because by default, it's 120 seconds and this is to long.
+            if len(partner.vat) > 5 and partner.vat[:2].lower() in stdnum_vat.country_codes: #TODO OCO les country codes font qu'on ne peut pas s'en passer, semble-t-il ... mais bon => vatnumber n'a pas ça ? Qu'on n'en garde qu'un seul des deux ... => de toute façon, ce check est-il réellement primordial ? > tester
```

Comment:
Shouldn't those comments be in English? Just saying 😄

Reply:
It is actually some development comment I made while working on this. It has been removed by another commit before I create the pull request. The commits of this branch will be squashed when merging it, so you should only pay attention to its current diff, and not its history, for this is the only thing that matters ;) The history of a development branch always contains some draft code, it's part of the usual dev process ;)

Reply:
Oh right, my bad there 😓